websites/Y/YouTube/dist/metadata.json

```diff
@@ -29,13 +29,51 @@
     "www.youtube.com",
     "studio.youtube.com"
   ],
-  "version": "4.2.7",
+  "version": "5.0.0",
```

Comment:
Why jump to 5?!?! Just use 4.3.0

Reply:
bruh he rewrote whole presence and you are saying jump to 4.3.0 xD

Reply:
And me rewrote my entire website 

![IMG_20201225_185658.jpg](https://user-images.githubusercontent.com/13917975/103138374-fca29a00-46e2-11eb-851b-a91fb7019bdc.jpg)

Reply:
This repo is about PreMiD Presences not your website.

Reply:
Still, it's a big change for the service the presence is made for.